ci/checks/copyright.py

```diff
@@ -38,6 +38,9 @@
     re.compile(r"[.]flake8[.]cython$"),
     re.compile(r"meta[.]yaml$")
 ]
+ExemptFiles = [
+    re.compile(r"_thirdparty")
```

Comment:
I would make this more specific so it only applies to the python folder. Should be `python/cuml/_thirdparty`/

Reply:
I think we'll need an ASAP fix, as thirdparty section changes can't really be committed right now. But I agree on making it more specific.